### ISSUE TITLE
[Drag & Drop blocks] Disable dragging when a screen reader is enabled

### ISSUE THREAD

**Issue by fluiddot** [wordpress-mobile/gutenberg-mobile#4819]
## Description

The dragging gesture can be performed when a screen reader (like VoiceOver or TalkBack) is enabled by the long-press gesture, however, we found the following issues:
- Dragging should be enabled by the command "Tap twice and hold" ([reference](https://dequeuniversity.com/screenreaders/voiceover-ios-shortcuts#other_commands)), not the long-press gesture.
- There's no indication about when a block is being dragged and dropped, nor the dropping location.

For these reasons, we should consider disabling the dragging until we fully support screen readers. Additionally, as the editor already provides support for moving a block via the up/down arrows, disabling the drag & drop shouldn't impact the user experience.

**Comment by fluiddot** [comment]
Closing this issue as it was addressed in https://github.com/WordPress/gutenberg/pull/40852.
